run.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
what about calling this file `toolbox.py` to keep "toolbox" in the invocation command?
that may clash with Python `toolbox` package directory though,
so maybe `toolbox` dir --> `toolbox_scripts`,
or `run.py` --> `run_toolboxy.py` or `toolbox_run.py` ?

Reply:
>that may clash with Python toolbox package directory though,

That's the reason I didn't do it

>so maybe toolbox dir --> toolbox_scripts,

I think I literally did exactly that, like, literally, this exact same name
But I didn't do it because I wanted to keep older scripts in that directory in their exact same position for backwards compatibility

>or run.py --> run_toolboxy.py or toolbox_run.py ?

Yeah that's possible.